resonate/models/handle.py

```diff
@@ -4,14 +4,16 @@
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
-    from collections.abc import Generator
+    from collections.abc import Callable, Generator
     from concurrent.futures import Future
 
 
 class Handle[T]:
-    def __init__(self, id: str, f: Future[T]) -> None:
+    def __init__(self, id: str, f: Future[T], should_subscribe: bool, cb: Callable[[str, Future[T]], None]) -> None:
```

Comment:
For simplicity let's just remove `should_subscribe`, we already dedup the call in the scheduler so making sending multiple commands should not hurt. 

For the cb, do you mind passing in a reference to the bridge itself instead?